### DISCUSSION TITLE
Is it ok that the training loss curve has no obvious change when started training backbone?

### DISCUSSION THREAD

**Discussion by LNoving** [STVIR/pysot#391] [General] [answered]
According to the code, when training, SiamRPN++ fix the backbone for the first 10 epochs and start training backbone in 11. 
In many cases, when a backbone changes from fixed to training, there will be a sudden change in the loss curve, generally sudden decrease. However, in my curve, there is no obvious change. So I wonder if there's something wrong in my training.
Can someone explain this? Or can someone offer some training loss curves as references? Thanks!

**Comment by ZhiyuanChen** [comment, marked as answer]
I didn't do the test, but a well pretrained backbone are expected to extract good features from the image. I can't see why there should be a sudden drop.

**Comment by LNoving** [comment]
> I didn't do the test, but a well pretrained backbone are expected to extract good features from the image. I can't see why there should be a sudden drop.

I see. Thanks for replying.
